src/grid/grid.component.ts

```diff
@@ -764,6 +764,18 @@ export class IgxGridComponent implements OnInit, OnDestroy, AfterContentInit, Af
         return true;
     }
 
+    /**
+     * Recalculates grid width/height. Should be ran when changing DOM elements dimentions manually that affect the grid.
+     */
+    public calculateGridSizes() {
```

Comment:
The current name suggests that the grid size is calculated and returned by the method. Instead the method reflows the grid, so `reflow()` is a better name.